<launch>   

  
  <node pkg="v4r_uvc" type="v4r_uvc_node" name="camera" >
    <param name="video_device" value="/dev/video1"/>
    <param name="show_camera_image" value="false"/>  
    <param name="convert_image" value="1"/>    
    <param name="camera_info_url" value="file://$(find v4r_uvc)/cfg/logitech.yml" />
    <param name="frame_id" value="camera"/>
    <param name="brightness" value="30"/>
    <param name="white_balance_temperature_auto" value="true"/>
  </node>
   
   <!-- undistore image -->
  <group ns="camera" > 
    <node pkg="image_proc" type="image_proc" name="image_proc" />
  </group>
  
</launch>
